### PR TITLE
chore(ci): temporarily disable OS X

### PR DESCRIPTION
**Pull request by mmarchini**
Refs: https://github.com/nodejs/llnode/issues/362

### CODE REVIEW

**Comment by mmarchini** [comment]
cc @nodejs/llnode

**Comment by joyeecheung** [comment]
It seems that `--abort-on-uncaught-exception` has regressed on Node.js v14 which is why the test is timing out (instead of aborting, the process exits with -1 normally)

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
             mirror: https://nodejs.org/download/nightly
           - version: 15.x
             mirror: https://nodejs.org/download/v8-canary
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
```
Comment (joyeecheung) [inline .github/workflows/push.yml:26]:
```suggestion
        # os: [ubuntu-latest, macos-latest]
        # Temporarily disable MacOS until
        # https://github.com/nodejs/node/issues/32981 is fixed
        os: [ubuntu-latest]
```

**Comment by mmarchini** [comment]
> It seems that `--abort-on-uncaught-exception` has regressed on Node.js v14 which is why the test is timing out (instead of aborting, the process exits with -1 normally)

Ah yes, that might cause v14 tests to fail, although I'm pretty sure llnode is broken on v14 since we bumped V8 recently (as in, after the last time we updated things on llnode).

**Comment by joyeecheung** [comment]
From https://github.com/nodejs/node/issues/32981 it seems the bug was introduced from v12.11.0. Is there any progress in the V8 upstream?

**Comment by mmarchini** [comment]
I don't think I opened an upstream bug :thinking: I'll check tomorrow and will open one if I haven't yet.

**Comment by mmarchini** [comment]
Although tests are passing on v12, so that issue should be unrelated (hopefully)

**Pull request merged**